requirements-base.txt

```diff
@@ -72,8 +72,8 @@ uwsgi>2.0.0,<2.1.0
 # We specifically pin to 3.5.0.4 to avoid the issue in
 # https://github.com/celery/billiard/issues/260. Once we upgrade to Celery 4.3+ we can
 # use a newer version of billiard that has a proper fix.
-billiard==3.5.0.4
-kombu==4.2.2.post1
+billiard>=3.5.0.4,<=3.6.3
+kombu>=4.2.2.post1,<=4.6.11
```

Comment:
Can we leave a comment in here?

Reply:
This will be fixed in the next couple days, so I don't think we need to

Reply:
Just need this out earlier, since on-prem releases tomorrow